applications/nrf5340_audio/src/audio/audio_datapath.c

```diff
@@ -918,7 +918,7 @@ int audio_datapath_init(void)
 	audio_i2s_blk_comp_cb_register(audio_datapath_i2s_blk_complete);
 	ctrl_blk.datapath_initialized = true;
 	ctrl_blk.drift_comp.hfclkaudio_comp_enabled = true;
-	ctrl_blk.pres_comp.pres_delay_us = DEFAULT_PRES_DLY_US;
+	ctrl_blk.pres_comp.pres_delay_us = CONFIG_AUDIO_MIN_PRES_DLY_US;
```

Comment:
```suggestion
	ctrl_blk.pres_comp.pres_delay_us = CONFIG_BT_AUDIO_PRESENTATION_DELAY_US;
```

Reply:
What is CONFIG_BT_AUDIO_PRESENTATION_DELAY_US as I can not find it?
This is set to the minimum for the device currently.

Reply:
It's the one you renamed to  BT_AUDIO_LOCAL_PRESENTATION_DELAY_US